src/invoker.ts

```diff
@@ -305,16 +305,22 @@ export class ErrorHandler {
   register() {
     process.on('uncaughtException', err => {
       console.error('Uncaught exception');
-      logAndSendError(err, latestRes, killInstance);
+      sendCrashResponse({err, res: latestRes, callback: killInstance});
     });
 
     process.on('unhandledRejection', err => {
       console.error('Unhandled rejection');
-      logAndSendError(err, latestRes, killInstance);
+      sendCrashResponse({err, res: latestRes, callback: killInstance});
     });
 
     process.on('exit', code => {
-      logAndSendError(new Error(`Process exited with code ${code}`), latestRes);
+      if (code) {
```

Comment:
With this check, it won't send an error response for code 0.

Reply:
That's the point. 0 represents a successful exit status, and thus should not produce an error.

What is expected?

Reply:
I _think_ it still needs to be reported to the supervisor.

Reply:
Oh, I see the error in my logic.
PTAL again.